releasenotes/notes/bump-go-to-1.14.7-e7ca99c97da374b5.yaml

```diff
@@ -0,0 +1,3 @@
+other:
+  - |
+    The Datadog Agent, Logs Agent and the system-probe are now built with ``Go 1.14.7``.
```

Comment:
In previous go version bumps, we used the following formulation: "All Agents binaries are now compiled with Go \`\`x.y.z\`\`", so we should use the same here (cf. discussions in https://github.com/DataDog/datadog-agent/pull/5167, https://github.com/DataDog/datadog-agent/pull/5804, https://github.com/DataDog/datadog-agent/pull/5815).

Reply:
Sure thing. I've updated it in the same time as the builders images.